src/app/annotationViewer/positionLine.js

```diff
@@ -5,121 +5,123 @@ angular
         return {
             restrict: "A",
             templateUrl: "annotationViewer/positionLine.tpl.html",
-            replace: true,
+            replace: false,
             scope: {
-                media: "<media",
-                audioData: "<audioData",
-                // TODO: remove this binding
-                imageClass: "<imageClass"
+                media: "=media",
+                audioData: "=audioData"
             },
-            link: function (scope, elements, attributes, controller) {
+            controller: ["$scope", "$element", "lodash", function (scope, element, _) {
 
-                // the dom element for the spectrogram image that is used to get
-                // the width to calculate the pixel offset for the position line
-                // TODO: remove the need to reference an image element.
-                var image;
+                var self = this;
+                self.containerWidth = 0;
+
+                self.updateContainerWidth = function () {
+                    var container = element[0].parentElement;
```

Comment:
we typically refer to jquery[ lite] elements with a `$` prefix - e.g. `$element`
conversely, native DOM elements have no prefix - e.g. `element`

Adjust names please